lib/boilerplate.js

```diff
@@ -62,6 +62,72 @@ module.exports = function (moduleName, options = {}) {
           };
         `
       }
+    },
+    a3: {
+      'create-module': {
+        moduleConfig: stripIndent`
+          module.exports = {
+            extend: '@apostrophecms/module',
+            init(self) {
+
+            }
+          };
+        `
+      },
+      'create-piece': {
+        pieceConfig: stripIndent`
+          module.exports = {
+            extend: '@apostrophecms/piece-type',
+            options: {
+              label: '${util.titleCase(moduleName.replace(/-/g, ' '))}',
+              // TODO: Update the plural label for editors
+              pluralLabel: '${util.titleCase(moduleName.replace(/-/g, ' '))}',
```

Comment:
Going to ask that you fix this first because as written it's just not a plural label. Should also reuse the work for `label` as a starting point.

Appending an "s" will do for this boilerplate generation, but f you want to add an optional way to specify it that's cool too.

Reply:
That's what Apostrophe will do for them, so I'll simply remove this.